INSTALL.md

```diff
@@ -73,17 +73,20 @@ module.exports = {
         use: [
           'style-loader',
           { loader: 'css-loader', options: { importLoaders: 1 } },
-          { loader: 'postcss-loader', options: {
-            ident: 'postcss',
-            plugins: () => [
-              postcssPresetEnv(/* pluginOptions */)
-            ]
-          } }
-        ]
-      }
-    ]
-  }
-}
+          {
+            loader: 'postcss-loader',
+            options: {
+              postcssOptions: {
+                ident: 'postcss',
```

Comment:
That will fail.

Reply:
Updated to use an array instead of a function. This is identical to my own setup:)

Reply:
It's the `ident` property, sorry. It will crash, because it does not validate as a proper config key.

Reply:
Oh, I wasn't aware they had removed it. Sorry for the trouble! (I can rebase if you want.)